src/stru_multi/4C_stru_multi_microstatic.hpp

```diff
@@ -134,11 +134,16 @@ namespace MultiScale
         const double dt);
 
     /*!
+    \brief Write runtime output (displacement, tangent-stiffness, material IDs)
```

Comment:
Maybe this is a good point for a general overview of the output (see my previous comment).